### ISSUE TITLE
Unresolved dependencies: de.heikoseeberger#sbt-header;3.0.2 not found

### ISSUE THREAD

**Issue by Abrasha** [sbt/sbt-header#133]
Environment details:
- sbt.version = 0.13.16
- java version "1.8.0_144"
Java(TM) SE Runtime Environment (build 1.8.0_144-b01)
Java HotSpot(TM) 64-Bit Server VM (build 25.144-b01, mixed mode)
- javac 1.8.0_144
- Scala code runner version 2.12.3 -- Copyright 2002-2017, LAMP/EPFL and Lightbend, Inc.

I am trying to add sbt-header to my project to keep copyright headers up to date, but I am unable to add this plugin to sbt.

The error I am getting:

    [error] (*:update) sbt.ResolveException: unresolved dependency: de.heikoseeberger#sbt-header;3.0.2: not found

Detailed message:

```
[info] Resolving de.heikoseeberger#sbt-header;3.0.2 ...
[warn] 	module not found: de.heikoseeberger#sbt-header;3.0.2
[warn] ==== typesafe-ivy-releases: tried
[warn]   https://repo.typesafe.com/typesafe/ivy-releases/de.heikoseeberger/sbt-header/scala_2.10/sbt_0.13/3.0.2/ivys/ivy.xml
[warn] ==== sbt-plugin-releases: tried
[warn]   https://repo.scala-sbt.org/scalasbt/sbt-plugin-releases/de.heikoseeberger/sbt-header/scala_2.10/sbt_0.13/3.0.2/ivys/ivy.xml
[warn] ==== local: tried
[warn]   /Users/aabramov/.ivy2/local/de.heikoseeberger/sbt-header/scala_2.10/sbt_0.13/3.0.2/ivys/ivy.xml
[warn] ==== public: tried
[warn]   https://repo1.maven.org/maven2/de/heikoseeberger/sbt-header_2.10_0.13/3.0.2/sbt-header-3.0.2.pom
[warn] ==== local-preloaded-ivy: tried
[warn]   /Users/aabramov/.sbt/preloaded/de.heikoseeberger/sbt-header/3.0.2/ivys/ivy.xml
[warn] ==== local-preloaded: tried
[warn]   file:////Users/aabramov/.sbt/preloaded/de/heikoseeberger/sbt-header_2.10_0.13/3.0.2/sbt-header-3.0.2.pom
[info] Resolving org.fusesource.jansi#jansi;1.4 ...
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	::          UNRESOLVED DEPENDENCIES         ::
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	:: de.heikoseeberger#sbt-header;3.0.2: not found
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
```

As an example I took this project: https://github.com/sbt/sbt-fresh
Maybe I need to add some special resolvers to my project?

Thank you.

**Comment by Abrasha** [comment]
Suddenly, the issue fixed by upgrading sbt version to 1.0.2.

So another question: how can I use this plugin with sbt version 0.13.16?
I am trying to add sbt-header plugin to [Scala main repository](https://github.com/scala/scala) in scope of https://github.com/scala/bug/issues/10531.

**Comment by hseeberger** [comment]
My fault, I forgot to cross-publish. Done now. Thanks for reporting.
